File: ClientSide/src/app/pages/students/models/studentGridModel.ts
export class StudentGridModel {
    
    public id: number;
    public code: string;
    public fullName: string;
    public facultyTitle: string;
    public studyProgrammeTitle: string;
    public learningForm: string;
    public studentLanguage: string;

    constructor(
        id: number,
        code: string,
        fullName: string,
        facultyTitle: string,
        studyProgrammeTitle: string,
        learningForm: string,
        studentLanguage: string,
    ) {
        this.id = id;
        this.code = code;
        this.fullName = fullName;
        this.facultyTitle = facultyTitle;
        this.studyProgrammeTitle = studyProgrammeTitle;
        this.learningForm = learningForm;
        this.studentLanguage = studentLanguage;
    }
}

File: ServerSide/SAMS.BusinessLogic/Models/Students/StudentGridModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SAMS.BusinessLogic.Models.Students
{
    public class StudentGridModel
    {
        public int Id { get; set; }
        public string Code { get; set; }
        public string FullName { get; set; }
        public string FacultyTitle { get; set; }
        public string StudyProgrammeTitle { get; set; }
        public int LearningForm { get; set; }
        public int StudentLanguage { get; set; }
    }
}
